.github/workflows/build-dev.yaml

```
@@ -85,16 +85,18 @@ jobs:
+        uses: aquasecurity/trivy-action@v0.35.0
         env:
           TRIVY_DB_REPOSITORY: "ghcr.io/aquasecurity/trivy-db,public.ecr.aws/aquasecurity/trivy-db"
         with:
           scan-type: "fs"
+          trivyignores: ".trivyignore"
           ignore-unfixed: true
           format: "table"
           exit-code: "1"
           vuln-type: "os,library"
           severity: "CRITICAL,HIGH"
+          trivyignores: ".trivyignore"
```

actionlint (1.7.11)

[error] key "trivyignores" is duplicated in "with" section. previously defined at line:93,col:11. note that this key is case insensitive

(syntax-check)